src/oc/storage/resources/entry.clj

```diff
@@ -175,11 +192,24 @@
   (vec (sort-by :created-at
     (db-common/read-resources conn table-name :topic-slug-board-uuid [[topic-slug board-uuid]]))))
 
+(schema/defn ^:always-validate get-comments-by-topic
```

Comment:
maybe the right name for this function is `get-interactions-by-topic` or is going to be filtered once we work on interactions?